### PR TITLE
fix: theme status

### PR DESCRIPTION
**Pull request by pzzyf**
When the theme is dark, the switch status does not change

## Summary by CodeRabbit

- **New Features**
	- Introduced dynamic theme adjustments that allow the sidebar and header to respond to mode changes, enhancing user interactivity.
- **Improvements**
	- Enhanced the component's functionality to better react to state changes, making for a smoother user experience.

### CODE REVIEW

File: packages/effects/layouts/src/widgets/preferences/blocks/theme/theme.vue [hunk 1]
```diff
@@ -48,6 +48,13 @@ function nameView(name: string) {
     }
   }
 }
+
+watch(modelValue, (newValue) => {
+  if (newValue === 'dark') {
```
Comment (likui628) [inline packages/effects/layouts/src/widgets/preferences/blocks/theme/theme.vue:53]: 不是已经禁用了吗

Comment (pzzyf) [reply packages/effects/layouts/src/widgets/preferences/blocks/theme/theme.vue:53]: image
image
浅色状态给状态改了，切回深色状态没变

Comment (likui628) [reply packages/effects/layouts/src/widgets/preferences/blocks/theme/theme.vue:53]: 本来就不能变，变了切回去不就不对了。

Comment (pzzyf) [reply packages/effects/layouts/src/widgets/preferences/blocks/theme/theme.vue:53]: 但是切回深色状态应该是true，现在也是不对的。如果再切到浅色状态还是原来的话，得再记录一下上次状态

**Pull request closed without merging**